stylesheets/_component.tab-help.scss

```diff
@@ -0,0 +1,101 @@
+$nav-color-light: #3399cc !default;
+$nav-color-dark:  #2d89b7 !default;
+
+.tab-help-container {
+
+    -webkit-backface-visibility: hidden;
+    -webkit-transform: translate3d(-100%, 0, 0);
+    backface-visibility: hidden;
+    background: $nav-color-dark;
+    box-shadow: inset 4px 0 0 darken($nav-color-dark, 5%);
+    color: $white;
+    display: table-cell;
+    height: 100%;
+    right: -600px;
+    position: absolute;
+    top: 0;
+    transform: translate3d(-100%, 0, 0);
+    vertical-align: top;
+    width: 300px;
+
+    .lt-ie10 & {
+        right: -300px;
+    }
+
+    .open-help & {
+        -webkit-transform: translate3d(0, 0, 0);
+        filter: progid:DXImageTransform.Microsoft.Alpha(Opacity=100);
+        opacity: 1;
+        transform: translate3d(0, 0, 0);
+        right: -300px;
+    }
+}
+
+// hide help button on mobile when no JS
+
+.tab-help  {
+    -webkit-transform-origin: 50% 0%;
+    -webkit-transform: translate3d(70%, 0, 0) scale3d(0.9, 0.9, 0.9);
+    -webkit-transition-delay: ease, 0s;
+    -webkit-transition: opacity 300ms 100ms, -webkit-transform 400ms ease;
```

Comment:
Could we not let autoprefixer take care of these via the 'proper' versions below?